ui/src/components/drafts/components/DraftEditorHeader.js

```diff
@@ -70,7 +51,27 @@ class DraftEditorHeader extends React.Component {
       return false;
     }
 
-    if (condition) {
+    this.props.formRef.current.submit();
+
+    // remove the general_title from the form validation
+    delete formData.general_title;
+
+    // consider that the form is firstly empty
+    let emptyCondition = true;
+
+    // iterate to the first and second children of the object
+    // if all of them are empty send an empty form notification
+    Object.values(formData).map(formItem => {
+      let allEmptyValues = Object.values(formItem).filter(
```

Comment:
Won't work when `formItem` is not of type `object`

example to reproduce:
CMS Analysis form -> Switch "reproducible mode" enable-disable -> When disable the value in `formItem` is `undefined` ===> This will through an error

Reply:
Great catch. My question now is why the SwitchWidget when is deactivated returns undefined and not false?

Reply:
Yes, this is another separate issue ( here #1551, just created ).

For this PR lets just fix to work with non-object values